src/CommunalHelperModule.cs

```diff
@@ -52,6 +52,8 @@ public class CommunalHelperModule : EverestModule {
             DreamBlockDummy.Load();
 
             CustomDreamBlock.Load();
+            Entities.ConnectedStuff.ConnectedTempleCrackedBlock.Load();
```

Comment:
Once you change the namespace for the `ConnectedTempleCrackedBlock` class, you can apply this change:
```suggestion
            ConnectedTempleCrackedBlock.Load();
```
(Same for the `ConnectedTempleCrackedBlock.Unload` call).
You can also move it up with the other connected-type entities if you'd like (looks a bit cleaner).
Yeah, this is just a small nitpick :)

Reply:
Updated. Did not move since there did not seem to be a clear grouping for it to go in (most of the other connected entities do not have a function called during Load() )

Reply:
No worries about it then. The `Load` & `Unload` calls can be moved afterwards anyway, during manual cleanup for example.